### ISSUE TITLE
Invalid regex escape when searching

### ISSUE THREAD

**Issue by CarsonF** [SeedCompany/cord-api-v3#1712]
## Current Behavior
```
Invalid Regex: Illegal/unsupported escape sequence near index 10
(?i).*BTL\Kenya.*
```

## Expected Behavior

The blackslash needs to be escaped for it to be valid regex.

Searching myself I see that the partner they were looking for is actually: "BTL/Kenya". They got the slash wrong anyways. 

It might be better to try to strip all control characters and replace with spaces then treat each word as separate terms. 

So for example:
```
.*[BTL|Kenya].*
```

Though that would result in more getting matched and since this is unweight it would probably be worse in certain cases.
i.e Searching for "Carson Full" could return "Carson Parish" before "Carson Full"

**Comment by CarsonF** [comment]
Actually split like I mentioned above would be better because currently "Carson Full" doesn't match anything anyways, because there is no one field with that value (first & last name are split)

**Comment by CarsonF** [comment]
Better yet use a text comparison function instead of regex. https://neo4j.com/labs/apoc/4.1/misc/text-functions/#text-functions-text-similarity

**Comment by CarsonF** [comment]
Fixed #2098
